tests/enterprise.rs

```diff
@@ -1,128 +0,0 @@
-#![cfg(feature = "enterprise-tests")]
```

Comment:
@001wwang just calling your attention to this in case you want to re-introduce them.

Reply:
Thanks for the heads up. Yea, these might make their way into OPW (in a different form) depending on how reporting unfolds.